Sources/_CShims/include/platform_shims.h

```diff
@@ -24,6 +24,10 @@
 #include <libkern/OSThermalNotification.h>
 #endif
 
+#if defined(_WIN32)
+typedef int mode_t;
```

Comment:
What does `mode_t` mean on Windows? Why would we want Foundation to use UNIX/POSIX APIs on Windows?